CMakeLists.txt

```diff
@@ -38,11 +38,11 @@ if(CMAKE_VERSION VERSION_LESS 3.0)
     project(microxrcedds_client C)
     set(PROJECT_VERSION_MAJOR 1)
     set(PROJECT_VERSION_MINOR 0)
-    set(PROJECT_VERSION_PATCH 0)
+    set(PROJECT_VERSION_PATCH 1)
     set(PROJECT_VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH})
 else()
     cmake_policy(SET CMP0048 NEW)
-    project(microxrcedds_client VERSION "1.0.0" LANGUAGES C)
+    project(microxrcedds_client VERSION "1.0.1" LANGUAGES C)
```

Comment:
Can the string on `"1.0.1"` be generated with the previous defines like `PROJECT_VERSION_MINOR` in order to be more *DRY*?

Reply:
Ok, I updated the minimum CMake required version and remove the dependency from version 2.8.12. Be careful because it was also updated in MicroCDR, so merge it before.

Reply:
Ok, updated thirdparty

Reply:
Resolved: this PR will point to the (https://github.com/eProsima/Micro-CDR/pull/24) MicroCDR commit.